conf/messages

```diff
@@ -215,10 +162,156 @@ agentsLandingPage.what-you-will-need-to-complete-the-return.p2=You may be asked
 
-nino.title=Enter your National Insurance number
-nino.helpText=It’s on your National Insurance card, benefit letter, payslip or P60.<br>For example, ‘QQ 12 34 56 C’.
-nino.invalid=Your National Insurance number must be entered as 9 alphanumeric characters. For example, ‘QQ 12 34 56 C’
-nino.generic.error=National Insurance number must be entered as 9 alphanumeric characters. For example, ‘QQ 12 34 56 C’
+
+#===================================================
+# SERVICE WIDE AND GENERIC - START
+#===================================================
+
+service.title=Report and pay Capital Gains Tax on UK property
+service.signOut=Sign out
+hmrc_branding=HM Revenue & Customs
+exitSurvey.linkText=Tell us how we can improve this service (takes 1 minute)
+
+label.beta=Beta
+
+button.startNow=Start now
+button.continue=Continue
+button.saveAndContinue=Save and continue
+button.confirmAndContinue=Confirm and continue
+button.createAccount=Create account
+button.resendEmail=Resend email
+button.findAddress=Find address
+button.tryAgain=Try again
+button.startAgain=Start again
+button.signIn=Sign in
+button.manageAccount=Manage account
+button.acceptAndSend=Accept and send
+button.payNow=Pay now
+button.accountHome=Account home
+button.clients.accountHome=Client’s account home
+
+generic.warning=Warning
+generic.day=Day
+generic.month=Month
+generic.year=Year
+generic.back=Back
+generic.yes=Yes
+generic.no=No
+generic.success=Success
+generic.errorPrefix=Error:
+generic.loss=loss
+generic.gain=gain
+generic.on=on
+generic.for=for
+yesNo.true=Yes
+yesNo.false=No
+
+#Months
+date.1=January
+date.2=February
+date.3=March
+date.4=April
+date.5=May
+date.6=June
+date.7=July
+date.8=August
+date.9=September
+date.10=October
+date.11=November
+date.12=December
+
+date.short.1=Jan
+date.short.2=Feb
+date.short.3=Mar
+date.short.4=Apr
+date.short.5=May
+date.short.6=Jun
+date.short.7=Jul
+date.short.8=Aug
+date.short.9=Sept
+date.short.10=Oct
+date.short.11=Nov
+date.short.12=Dec
+
+cya.change=Change
+cya.delete=Delete
+cya.remove=Remove
+cya.add=Add
+
+unit-input.assistive.pounds=do not enter the pound sign
+unit-input.assistive.percent=do not enter the percent sign
+
+error.summary.heading=There’s a problem
+
+signed-out.title=You’ve signed out
+signed-out.p1=You’ve signed out of your Capital Gains Tax on UK property account.
+signed-out.p2=You can <a href="{0}">sign in</a> to your account.
+signed-out.button=Go to GOV.UK
+
+timed-out.title=For your security, we signed you out
+timed-out.p1=This was because you were inactive for too long.
+timed-out.p2=To access your Capital Gains on UK property account, you’ll need to sign in.
+timed-out.button=Sign in
+
+recruitment-banner.message=Help improve GOV.UK
+recruitment-banner.dismiss=No thanks
+recruitment-banner.link=Get involved in making Government services better (opens in a new tab)
+
+subsequentReturnExit.title=You’ll need to contact HMRC
```

Comment:
we're not likely to need this particular page translated - we'll have subsequent returns (hopefully) before we go live. We could send it to them anyway though just in case

Reply:
We can't remove it from messages file until we stop using it so I put it into the new TRANSLATION NOT REQUIRED section at the top along with landing page stuff